my-app/src/MapView.jsx

```diff
@@ -109,13 +112,13 @@ const MapView = () => {
           mapContainerClassName="MapContainer"
           center={center}
           zoom={13}
-          onZoomChanged={() => { 
+          onZoomChanged={() => {
             if (mapRef.current) {
               const newZoom = mapRef.current.getZoom();
-              setOverlapRadius(zoomToOverlapRadius(newZoom));
+              setZoomLevel(newZoom);
```

Comment:
New coding lesson: Instead of doing this in two lines, you could also do the following:

`setZoomLevel(mapRef.current.getZoom())`

Reply:
Okay ! I will look into fixing that